editor.planx.uk/src/components/Header/Sections/ProgressBar.tsx

```diff
@@ -27,10 +27,14 @@ const Root = styled(LinearProgress)(({ theme }) => ({
 }));
 
 export const ProgressBar: React.FC = () => {
-  const { completed, current } = useStore((state) =>
+  const progress = useStore((state) =>
     state.getSectionProgress(),
   );
 
+  if (!progress) return;
```

Comment:
This is just a type guard - the parent `SectionNavBar` will already be hidden if the flow does not use sections.